python/tank/platform/__init__.py

```diff
@@ -10,7 +10,7 @@
 
 
 # Engine management
-from .engine import start_engine, current_engine, get_engine_path, find_app_settings
+from .engine import start_engine, current_engine, get_engine_path, find_app_settings, _restart_engine
```

Comment:
I was actually planning to merge it myself, as the ticket mentionned. No worries for you. :)